flow360/component/simulation/models/surface_models.py

```diff
@@ -131,6 +131,27 @@ class Pressure(SingleAttributeModel):
     value: PressureType.Positive = pd.Field(description="The static pressure value.")
 
 
+class SlaterPorousBleed(Flow360BaseModel):
+    """
+    :class:`SlaterBleedPressure` class to specify the static pressure of the Slater bleed model
+    `Outflow` boundary condition via :py:attr:`Outflow.spec`.
+
+    Example
+    -------
+
+    >>> fl.SlaterPorousBleed(value = 1.01e6 * fl.u.Pa)
+
+    ====
+    """
+
+    type_name: Literal["SlaterPorousBleed"] = pd.Field("SlaterPorousBleed", frozen=True)
+    # pylint: disable=no-member
+    static_pressure: PressureType.Positive = pd.Field(description="The static pressure value.")
+    porosity: float = pd.Field(
+        gt=0, le=1, description="The porosity of the bleed region. Range [0,1]."
```

Comment:
No need for hand-written range. Documentation will handle it.
```suggestion
        gt=0, le=1, description="The porosity of the bleed region."
```

Reply:
Do we need default values for porosity or static_pressure?